packages/markdown-docx/src/CiceroMarkToOOXMLTransformer.js

```diff
@@ -14,23 +14,73 @@
 
 'use strict';
 
+const { TEXT_RULE, EMPHASIS_RULE } = require('./CiceroMarkToOOXMLRules');
+const { wrapAroundDefaultDocxTags } = require('./CiceroMarkToOOXMLHelpers');
+
+const definedNodes = {
+    computedVariable: 'org.accordproject.ciceromark.ComputedVariable',
+    heading: 'org.accordproject.commonmark.Heading',
+    item: 'org.accordproject.commonmark.Item',
+    list: 'org.accordproject.commonmark.List',
+    listBlock: 'org.accordproject.ciceromark.ListBlock',
+    paragraph: 'org.accordproject.commonmark.Paragraph',
+    softbreak: 'org.accordproject.commonmark.Softbreak',
+    text: 'org.accordproject.commonmark.Text',
+    variable: 'org.accordproject.ciceromark.Variable',
+    emphasize: 'org.accordproject.commonmark.Emph',
+};
+
+let globalOOXML = '';
```

Comment:
Why did you move this outside the class? This should be an instance variable inside the constructor.

Reply:
@algomaster99 
[Here](https://github.com/accordproject/markdown-transform/pull/400), you suggested it move out of the class. EsLint was unable to parse this as a class variable.

Reply:
@algomaster99 Made the other suggested changes.

Reply:
Class variables and instance variables are different. Instance variables are specific to the object of the class. They are accessed using the `this` keyword. [Example](https://github.com/accordproject/markdown-transform/blob/master/packages/markdown-cicero/lib/CiceroMarkTransformer.js#L47).

Class variables or static variables are common for all objects of the class. They don't even require you to instantiate a class to access them.

Reply:
I think I misunderstood this one :sweat_smile: . I will revert it and push.

Reply:
@algomaster99 Done